File: GamerHub_Backend/Entities/DetallesCompra.cs
using Microsoft.Build.Framework;

namespace GamerHub_Backend.Entities
{
    public class DetallesCompra
    {
        public int Id { get; set; }
        [Required]
        public int IdOrden { get; set; }
        public OrdenCompra? OrdenCompra { get; set; }
        [Required]
        public int IdProducto { get; set; }
        public Producto? Producto { get; set;} 
        [Required]
        public int Cantidad { get; set; }
        [Required]
        public decimal Precio { get; set; }
        
        
    }
}

File: GamerHub_Store/src/app/interface/DetallesCompra.ts
import { OrdenCompra } from "./OrdenCompra";
import { Producto } from "./Producto";

export interface DetallesCompra {
    id?: number;
    idOrden: number;
    ordenCompra?: OrdenCompra;
    idProducto: number;
    producto?: Producto;
    cantidad: number;
    precio: number;
}
